data_diff/databases/duckdb.py

```diff
@@ -167,12 +167,16 @@ def select_table_schema(self, path: DbPath) -> str:
         database, schema, table = self._normalize_table_path(path)
 
         info_schema_path = ["information_schema", "columns"]
+
         if database:
             info_schema_path.insert(0, database)
+            dynamic_database_clause = f"'{database}'"
```

Comment:
This is a dynamic clause because it accepts multiple input configurations:
- database.schema.table
- schema.table
- table (this will error out as expected)